html/index.html.haml

```diff
@@ -81,6 +81,20 @@
           = render_scope scope
 
         .hidden#inspection_details
+          - if @description[:environment].attributes.include?("system_type")
```

Comment:
This check also needs to be adapted. See the other comment for more information.